config.py

```diff
@@ -14,4 +20,39 @@ class Config(object):
     SERVICE_URL = os.environ.get("SERVICE_URL", "")
     SESSION_COOKIE_HTTPONLY = True
     SESSION_COOKIE_SECURE = True
-    DEFAULT_AWS_PROFILE = os.getenv("DEFAULT_AWS_PROFILE")
+
+    @property
```

Comment:
we can probably further organise our configuration into a Production config which calls env vars from paramstore, another which just reads from env vars etc. but for now this works and I didnt want to make this PR any bigger.